libs/components/src/lib/switch/switch.scss

```
@@ -0,0 +1,69 @@
+@use '../../../../../dist/libs/tokens/scss/tokens.constants' as constants;
+
+@use '../../../../shared/src/lib/sass/mixins/connotation/config' with (
+	$connotations: accent cta alert,
+	$shades: primary primary-text contrast,
+	$default: accent,
+);
+@use '../../../../shared/src/lib/sass/mixins/connotation' as connotation;
+
+@use '../../../../shared/src/lib/sass/mixins/appearance/config' as appearance-config with (
+	$appearances: filled ghost,
+	$states: idle disabled readonly error success,
+	$default: filled,
+);
+@use '../../../../shared/src/lib/sass/mixins/appearance' as appearance;
+
+.control {
+	@include connotation.connotation;
+	@include appearance.appearance;
+
+	display: inline-flex;
+}
+
+.switch {
+	width: 36px;
+	height: 20px;
+	border: 1px solid var(constants.$vvd-color-neutral-500);
+	border-radius: 10px;
+
+	.disabled & {
+		border-color: var(constants.$vvd-color-neutral-200);
+	}
+
+	[aria-checked="true"] & {
+		background-color: var(#{appearance.get-appearance-token(fill)});;
+	}
+}
+
+.checked-indicator {
+	--checked-indicator-size: 12px;
+
+	position: absolute;
+	width: var(--checked-indicator-size);
+	height: var(--checked-indicator-size);
+	margin-top: calc((20px - var(--checked-indicator-size)) / 2);
```

```suggestion
	margin-top: calc((18px - var(--checked-indicator-size)) / 2);
```
because I added box-sizing